app/Http/Livewire/Components/Header.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire\Components;
+
+use App\Http\Livewire\Traits\InteractWithModal;
+use App\Models\Configs;
+use Illuminate\Support\Facades\Config;
+use Livewire\Component;
+
+class Header extends Component
+{
+	use InteractWithModal;
+
+	/**
+	 * @var string
+	 */
+	public $title = '';
+
+	/**
+	 * @var string
+	 */
+	public $mode;
+
+	public function mount(?string $mode = 'albums', ?string $title = null)
```

Comment:
```suggestion
	public function mount(?string $mode = 'albums', ?string $title = null): void
```

 - Add a PHPdoc comment what this method does. Why it is called "mount"? What does it mount?
 - Do we really need the first parameter to be nullable given that a default value is provided?
 - What kind of mode do we speak about? Can we make it more specific, i.e. `accessMode`, `modeOfExplosion`? What are the admissible values?
 - Please avoid hard-coded magic constants. If there are only a few valid values (I assume so), then introduce constants, i.e. `const MODE_ALBUMS = 'albums'`.